docs/gcloud-auth.rst

```diff
@@ -0,0 +1,310 @@
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+GCloud Auth
+-----------
+
+For the majority of cases, authentication with the ``gcloud`` library
+will **"just work"**. Credentials are loaded implicitly from the environment
+where the code is running and **no code** is required to enable authentication
+within your code.
+
+The core :mod:`gcloud.credentials` module provides helper methods
+for any account type needed.
+
+A production application should **use a service account**, but you may
+wish to use other methods when first getting started with the ``gcloud``
+library.
+
+=============
+Basic Example
+=============
+
+After downloading a JSON key for a service account (see `Service Accounts`_),
```

Comment:
It seems to me that, you always have to create a service account and download the key, even on App Engine or Compute Engine instances.

Can you make it clear that you don't have to create a service account on App Engine prod and GCE instances?